core/App/screens/ProofRequest.tsx

```diff
@@ -124,7 +124,9 @@ const ProofRequest: React.FC<ProofRequestProps> = ({ navigation, route }) => {
     setLoading(true)
     const retrieveCredentialsForProof = async (proof: ProofRecord) => {
       try {
-        const credentials = await agent.proofs.getRequestedCredentialsForProofRequest(proof.id)
+        const credentials = await agent.proofs.getRequestedCredentialsForProofRequest(proof.id, {
```

Comment:
should proof requests with revoked credentials be in a separate PR?

Reply:
Just wondering if we're putting too much in one PR

Reply:
Styling revoked credentials are part of the feature. I could leave it out but then wed have to wait on another PR to get this feature fully tested.